frontend/src/scenes/persons/Person.tsx

```diff
@@ -87,6 +87,21 @@ function PersonCaption({ person }: { person: PersonType }): JSX.Element {
                 <span className="text-muted">First seen:</span>{' '}
                 {person.created_at ? <TZLabel time={person.created_at} /> : 'unknown'}
             </div>
+            <div>
+                <span className="text-muted">Merge restrictions:</span> {person.is_identified ? 'applied' : 'none'}
+                <Link to={'https://posthog.com/docs/data/identify'}>
+                    <Tooltip
+                        title={
+                            <>
+                                {person.is_identified ? <strong>Cannot</strong> : 'Can'} be used as `$anon_distinct_id`
+                                or `alias_id`, click for more info.
```

Comment:
I thought I commented this already but couldnt find my previous comment:

Is it worth including `$anon_distinct_id` here? This is for using as an anonymous ID right? This is not super common right? In fact, I wasn't even aware you could do it.

When would our users want to do it? If I recall correctly, we dont mention this in our docs anywhere?

Reply:
https://github.com/search?q=repo%3APostHog%2Fposthog.com%20anon_distinct_id&type=code but sure I can keep only `alias_id` can you look into the experiments doc https://posthog.com/tutorials/experiments and update that too. The how ingestion works should mention it and be unchanged probably.